processing/src/main/java/io/druid/segment/StringDimensionIndexer.java

```diff
@@ -539,7 +546,70 @@ public int lookupId(String name)
   }
 
   @Override
-  public Object convertUnsortedEncodedArrayToActualArrayOrList(int[] key, boolean asList)
+  public LongColumnSelector makeLongColumnSelector(
+      IncrementalIndexStorageAdapter.EntryHolder currEntry, IncrementalIndex.DimensionDesc desc
+  )
+  {
+     return ZeroLongColumnSelector.instance();
```

Comment:
How valid is it to return zero selector here? Maybe try to convert from String to number?